### PR TITLE
Exasol table definition

### PR DESCRIPTION
**Pull request by zajca**
KBC: https://github.com/keboola/connection/pull/3334

Before asking for review make sure that:

- [x] New client method(s) has tests
- [x] Apiary file is updated
- [x] You declared if there is a BC break or not (will affect next release of a client)

---

### CODE REVIEW

File: tests/Backend/Synapse/TableDefinitionOperationsTest.php [hunk 1]
```diff
@@ -59,30 +59,30 @@ private function createTableDefinition()
...
     public function testRemovePrimaryKeyOnTypedTable()
     {
-        $this->expectExceptionMessage("Not implemented for typed synapse tables");
+        $this->expectExceptionMessage("Not implemented for typed tables");
         $this->_client->removeTablePrimaryKey($this->tableId);
     }
 
     public function testCreateSnapshotOnTypedTable()
     {
-        $this->expectExceptionMessage("Not implemented for typed synapse tables");
```
Comment (jirkasemmler) [inline tests/Backend/Synapse/TableDefinitionOperationsTest.php:85]: trochu BC break, ale asi naprosto minimalni

File: apiary.apib [hunk 2]
```diff
@@ -1564,11 +1564,11 @@ Request is handled asynchronously which means that the create request is created
     + columns[] (required) - definition of table columns
        + name (required, string) - column name
        + definition (object) - column settings
-            + type (required, string) - datatype of this column, choose one from https://bit.ly/3uZjN60
+            + type (required, string) - datatype of this column, choose build in types for used backend
```
Comment (jirkasemmler) [inline apiary.apib:1567]: klidne bych tam dal link na ty Synapse i Exasol typy

**Review by jirkasemmler — COMMENTED** [review]
asi malickosti, pustim az pofixujeme Connection

File: tests/Backend/Synapse/TableDefinitionOperationsTest.php (hunk 1 above)
Comment (zajca) [reply tests/Backend/Synapse/TableDefinitionOperationsTest.php:85]: To bych asi vůbec neřešil je to experimental a ještě not implemented string message

File: apiary.apib (hunk 2 above)
Comment (zajca) [reply apiary.apib]: přihodil sem

**Review by jirkasemmler — APPROVED** [review]
lgtm, snad priste bych jeste prepsal ty $this->assert na self::assert at to nesviti ve stormu

**Pull request merged**